### PR TITLE
chore: update network skeleton docs

### PR DESCRIPTION
**Pull request by kamuik16**
## Proposed Changes
Updates Docs to include the reference to the PRs to the latest network skeleton upgrade (24). 

cc @rjan90

Also there were two Filecoin-FFI section, so I removed one of those, lmk if I'm wrong.

### CODE REVIEW

**Review by BigLep — COMMENTED** [review]
Good catch on the duplicate filecoin-ffi section.  I assume that was me...  Thanks for fixing.  

I'll let @rjan90 respond on if there is any reason not to update the doc to point to nv24 (more recent) example than the nv23 one.  I'm not aware of reason, but I think it would come down to if nv24 steps didn't follow the desired process for some reason.

**Comment by kamuik16** [comment]
> Good catch on the duplicate filecoin-ffi section. I assume that was me... Thanks for fixing.
> 
> I'll let @rjan90 respond on if there is any reason not to update the doc to point to nv24 (more recent) example than the nv23 one. I'm not aware of reason, but I think it would come down to if nv24 steps didn't follow the desired process for some reason.

I had done the complete nv24 skeleton upgrade by following all the steps only, but happy to wait for @rjan90 approval.

**Review by rjan90 — APPROVED** [review]
Thanks! Looks good to me

**Pull request merged**
